packages/scaffold-config/src/frameworks.ts

```diff
@@ -267,17 +266,55 @@ export const WIZARD_FRAMEWORKS = [
     category: 'library',
     name: 'Svelte.js',
     detectors: [dependencies.WIZARD_DEPENDENCY_SVELTE],
-    supportedBundlers: [dependencies.WIZARD_DEPENDENCY_WEBPACK, dependencies.WIZARD_DEPENDENCY_VITE],
-    dependencies: (bundler: WizardBundler['type'], projectPath: string): Promise<DependencyToInstall[]> => {
-      return Promise.all([
-        getBundlerDependency(bundler, projectPath),
-        isDependencyInstalled(dependencies.WIZARD_DEPENDENCY_SVELTE, projectPath),
-      ])
+    supportedBundlers: ['webpack', 'vite'],
+    dependencies: (bundler: WizardBundler['type']): Cypress.CypressComponentDependency[] => {
+      return [
+        getBundler(bundler),
+        dependencies.WIZARD_DEPENDENCY_SVELTE,
+      ]
     },
     codeGenFramework: 'svelte',
     glob: '*.svelte',
     mountModule: mountModule('cypress/svelte'),
     supportStatus: 'alpha',
     componentIndexHtml: componentIndexHtmlGenerator(),
   },
-] as const
+]
+
+export type ComponentFrameworkDefinition = Omit<Cypress.ResolvedComponentFrameworkDefinition, 'dependencies'> & {
+  dependencies: (bundler: WizardBundler['type']) => Cypress.CypressComponentDependency[]
```

Comment:
```suggestion
  dependencies: (bundler: 'webpack' | 'vite') => Cypress.CypressComponentDependency[]
```

Hovering over the type definition for `ThirdPartyComponentFrameworkDefinition` shows `WizardBundler['type']`. We should be more explicit for user facing types